lib/config/configuration.go

```diff
@@ -1815,6 +1818,9 @@ func Configure(clf *CommandLineFlags, cfg *servicecfg.Config, legacyAppFlags boo
 	}
 
 	if clf.ApplyOnStartupFile != "" {
+		if !utils.FileExists(clf.ApplyOnStartupFile) {
+			return trace.NotFound("failed to bootstrap cluster configuration: apply on startup file specified '%s' does not exist", clf.ApplyOnStartupFile)
+		}
 		resources, err := ReadResources(clf.ApplyOnStartupFile)
```

Comment:
This pattern is generally frowned upon. You have to handle the file doesn't exist error from `ReadResources` anyway (it could be removed/renamed after you check for its existence and before you try to open it.

Can we remove this explicit check and instead check whether the error coming back from `ReadResources` indicates that the file doesn't exist?